scripts/objc-test.sh

```diff
@@ -16,9 +16,17 @@ function cleanup {
     WATCHMAN_LOGS=/usr/local/Cellar/watchman/3.1/var/run/watchman/$USER.log
     [ -f $WATCHMAN_LOGS ] && cat $WATCHMAN_LOGS
   fi
+  [ $SERVER_PID ] && kill -9 $SERVER_PID
 }
 trap cleanup EXIT
 
+# Run the packager process directly
+node ./local-cli/cli.js start &
```

Comment:
Does it conflict with running the script manually?
Developers do that when they break tests